assignments/prototypes.js

```diff
@@ -32,6 +59,23 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+function Humanoid(atttributes){
+  GameObject.call(this,atttributes);
+  CharacterStats.call(this,atttributes);
+  this.team=atttributes.team;
+  this.weapons=atttributes.weapons;
+  this.language=atttributes.language;
+  
+}
+Humanoid.prototype=Object.create(GameObject.prototype);
+Humanoid.prototype=Object.create(CharacterStats.prototype);
```

Comment:
You don't have to use `Object.create` with `GameObject.prototype` either - you only need to create the object once, and `CharacterStats` was already created from `GameObject` earlier.